packages/wormhole-sdk/src/domains.ts

```diff
@@ -28,6 +28,8 @@ export const DEFAULT_RPC_URLS = {
 }
 export type DomainId = keyof typeof DEFAULT_RPC_URLS
 
+export type DomainDescription = DomainId | 'arbitrum-testnet' | 'optimism-testnet' //| 'arbitrum' | 'optimism'
```

Comment:
i am still wondering shouldn't we just drop the full names... the problem with "full names" is that they interfaces might be different and pointing new SDK to old bridge wouldn't work anyway...

Reply:
Do you mean making actual domain ids like `KOVAN-SLAVE-OPTIMISM-1` non-transparent to the client? I guess it would make sense if the sdk also takes care of picking the right domain based on amount withdrawn, debt ceiling utilization and fees. I would still want to allow the client to present this choice to the user at the UI level though. Maybe as a trade-off, the low-level WormholeBridge functions could use exact domain ids while the high level wrappers would only use descriptions like `arbitrum-testnet` and `optimism-testnet`?

Reply:
My comment above assumes that there can be more than one active domain ID on a particular network (for example one Optimism domain ID with a large debt ceiling, for which oracles require L1 confirmations and another Optimism domain ID with a smaller debt ceiling for which oracles trust the sequencer). If we don't think this is a likely setup, then I agree that there is no point in exposing the full domain ids.